src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -240,14 +247,13 @@ public partial class IPAddress
         public static short NetworkToHostOrder(short network) { throw null; }
         public static int NetworkToHostOrder(int network) { throw null; }
         public static long NetworkToHostOrder(long network) { throw null; }
+        public static System.Net.IPAddress Parse(System.ReadOnlySpan<char> ipSpan) { throw null; }
         public static System.Net.IPAddress Parse(string ipString) { throw null; }
-        public static System.Net.IPAddress Parse(ReadOnlySpan<char> ipString) { throw null; }
         public override string ToString() { throw null; }
-        public bool TryFormat(Span<char> destination, out int charsWritten) { throw null; }
+        public bool TryFormat(System.Span<char> destination, out int charsWritten) { throw null; }
+        public static bool TryParse(System.ReadOnlySpan<char> ipSpan, out System.Net.IPAddress address) { throw null; }
```

Comment:
Breaking change: It looks like we shipped `TryParse(System.ReadOnlySpan<char> ipString, ...)`. It should have been `ipSpan`.